src/__tests__/builder.spec.ts

```diff
@@ -17,18 +14,6 @@ describe('Builder', () => {
       b.addMarkdown(markdown);
       expect(b.toString()).toEqual(`${markdown}\n`);
     });
-
-    it('correctly parses stoplight markdown tabs', () => {
```

Comment:
Any reason for the removal of these two tests?

Reply:
The equivalent moved to the `reader.spec.ts` test suite (see line 246 onwards). Obv not exactly the same since doesn't invoke the builder `addMarkdown` method etc, but I figured testing the `addMarkdown` specifically feels a bit overkill. As mentioned in original PR description, in general there is an uncomfortable amount of overlap in reader/builder/frontmatter/parse/stringify that makes deciding where tests go etc kind of annoying. 

Anyways - rant over :). Do you think we should isolate and test something specific w the `addMarkdown` method?

Reply:
Nah, feels okay. Was simply curious to get to know why.